integration-tests/rest-client-reactive-stork/pom.xml

```diff
@@ -121,6 +116,18 @@
             </resource>
         </resources>
         <plugins>
+            <plugin>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <configuration>
+                    <annotationProcessorPaths>
+                        <path>
+                            <groupId>io.smallrye.stork</groupId>
+                            <artifactId>stork-configuration-generator</artifactId>
+                            <version>2.7.0</version>
```

Comment:
The hardcoded version number here is concerning, plus it seems useless. The Maven compiler plugin [documentation](https://maven.apache.org/plugins/maven-compiler-plugin/compile-mojo.html) says that this is resolved from `<dependencyManagement>` automatically?

Reply:
I beat you to it :).